pkg/orm/util.go

```diff
@@ -0,0 +1,364 @@
+package metautil
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+	"time"
+
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+
+	cerrors "github.com/hanfei1991/microcosm/pkg/errors"
+	"github.com/hanfei1991/microcosm/pkg/orm/model"
+)
+
+// TODO: retry and idempotent??
+// TODO: context control??
+
+const (
+	DefaultConnMaxIdleTime = time.Duration(30)
+	DefaultConnMaxLifeTime = time.Duration(18600)
```

Comment:
How is the default value `18600` chosen, it is 5h 10m.

Reply:
an enough long lifetime for the conn

Reply:
- The unit of time.Duration is Nanosecond, these two default values are not seconds here
- Better to use a more meaningful default value to prevent other people to be confused